google-cloud-datastore/pom.xml

```diff
@@ -38,12 +38,17 @@
       <version>1.3.0</version>
       <exclusions>
         <exclusion>
-          <artifactId>guava-jdk5</artifactId>
           <groupId>com.google.guava</groupId>
+          <artifactId>guava-jdk5</artifactId>
         </exclusion>
       </exclusions>
     </dependency>
     <dependency>
+      <groupId>com.google.api.grpc</groupId>
+      <artifactId>grpc-google-common-protos</artifactId>
+      <version>${generatedProto.version}</version>
+    </dependency>
```

Comment:
copy&pasting random xml from one pom.xml to another and doing "mvn install" to check if it works - this is pretty much what my maven skill is =)

But seriously speaking, I agree with @pongad.

The xsd schema for dependency <exclusion> element does not even support <version> sub-element, so I assume that exclusion should be used to exclude dependency completely, but not to modify its version. For our problem - we still need the dependency to be present, but with different version.

Modifying datastore-v1-protos-1.3.0.pom, and figuring out why this exists in that file:

```
    <!--
      Must depend on the same version of com.google.protobuf:protobuf-java
      that we do.
    -->
    <project.grpc-google-common-protos.version>0.1.0</project.grpc-google-common-protos.version>
```
 would be the right fix. But for now @pongad's fix seems good enough.